templates/epilepsy12/partials/organisation/cohort_card.html

```diff
@@ -1,58 +1,94 @@
 <div class="ui card">
-    <div class='cohorts_dates_grid'>
```

Comment:
I think it's worth splitting out the UI changes from the backend changes while we continue the discussion in #1120

Reply:
Sorry yes included here as felt this was quite urgent. I am sitting next to Amani ATM at a diabetes day so will ask her

Reply:
Yes there's also an email thread ongoing about how the dashboard should render during the grace period - lets wait to hear back on that before proceeding

Reply:
retrospective note for the trail. conversation with Nikky - suggests keep the three tiles but emphasise the active cohort